CHANGELOG.md

```diff
@@ -19,7 +19,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed a bug in `Databricks.create_table_from_pandas()`. The function that converts column names to snake_case was not used in every case. (#672)
 - Added `howto_migrate_sources_tasks_and_flows.md` document. This document will assist the DEs with the viadot 1 -> viadot 2 migration process.
 - `RedshiftSpectrum.from_df()` now automatically creates a folder for the table if not specified in `to_path`
-
+- Fixed a bug in `Databricks.create_table_from_pandas()`. DataFrame columns are now automatically converted using the `utils._cast_df_cols()`. 
```

Comment:
The bug is explained in the issue and the explanation of the solution is in the PR; here only mention the change; eg. "`Databricks.to_df()` now automatically casts DataFrame types. (#701)"

Reply:
[📝 Updated docstings and added entry to CHANGELOG.md](https://github.com/dyvenia/viadot/pull/701/commits/b8ec2278fff1a3cbb2c17cdb02c744662261fce7)